packages/@okta/vuepress-site/docs/guides/customize-tokens-returned-from-okta/create-app-profile-attribute/index.md

```diff
@@ -0,0 +1,96 @@
+
+To add a custom claim:
+
+1. In the Developer Console, navigate to **API**, and then **Authorization Servers**.
+
+2. Select the name of the Custom Authorization Server where you want to add the claim, and then click **Claims**.
+
+3. Click **Add Claim**, and then configure the claim settings:
+
+    > **Note:** For more information on these fields, see <GuideLink link="../add-custom-claim">Add a custom claim to a token</GuideLink>.
+
+    * **Name** &mdash; Enter a name for the claim, such as **applabel**.
+
+    * **Include in token type** &mdash; Leave the default of **Access Token**.
+
+    * **Value type** &mdash; Leave the default of **Expression** to define the claim by an Expression written in Okta Expression Language.
+
+    * **Value** &mdash; This option appears if you chose **Expression**. For this example, enter `app.profile.label`, which is referencing the app Profile attribute that you want to include in the claim.
+
+    > **Note:** You can validate that your expression returns the results expected using the **Token Preview** tab.
+
+    * **Disable claim** &mdash; Leave this clear for this example.
+
+    * **Include in** &mdash; Leave **Any scope** selected for this example.
+
+4. Click **Create**.
+
+5. (Optional) Confirm that your custom claim was successfully added by retrieving a list of all claims from your authorization server, including the custom claims, using the `/claims` endpoint. See <GuideLink link="../add-custom-claim/#verify-the-custom-claim">Verify the custom claim</GuideLink>.
+
+## Request a token that contains the custom claim
+
+In this example, the service application's `token_endpoint_auth_method` was set to `client_secret_post` when we created the app above. Include both the `client_id` and the `client_secret` values as additional parameters in the POST request body to your Custom Authorization Server's `/token` endpoint.
+
+```BASH
+curl -v -X POST \
+-H "Content-type:application/x-www-form-urlencoded" \
+"https://${yourOktaDomain}/oauth2/default/v1/token" \
+-d "client_id={client_id}&client_secret={client_secret}&grant_type=client_credentials"
+```
+
+If the credentials are valid, the access token is sent in the response:
+
+```JSON
+{
+    "access_token": "eyJhbG[...]1LQ",
+    "token_type": "Bearer",
+    "expires_in": 3600,
+    "scope": "customDefaultScope"
+}
+```
+
+> **Note:** When you [create a custom scope](/docs/guides/customize-authz-server/create-scopes/) in your Custom Authorization Server, you can select **Set as a default scope** to allow Okta to grant authorization requests to apps that don't specify scopes on an authorization request. If the client omits the scope parameter in an authorization request, Okta returns all default scopes in the access token that are permitted by the access policy rule. Depending on which flow you are using, it might also allow you to exclude the scope parameter from your token request.
```

Comment:
IMO we shouldn't be encouraging people to use the default scope feature, we should probably omit this note and request a scope in the example token request